src/site/content/en/blog/requestvideoframecallback-rvfc/index.md

```diff
@@ -0,0 +1,210 @@
+---
+title: |
+  The requestVideoFrameCallback (rVFC) API
+subhead: |
+  Perform efficient per-video-frame operations on video.
+authors:
+  - thomassteiner
+date: 2020-06-29
+updated: 2020-06-29
+hero: hero.jpg
+alt: Film roll.
+description: |
+  The requestVideoFrameCallback method allows web authors to register a callback,
+  which runs in the rendering steps when a new video frame is sent to the compositor.
+tags:
+  - blog
+  - media
+  - capabilities
+---
+There's a new Web API on the block, defined in the
+[HTMLVideoElement.requestVideoFrameCallback()](https://wicg.github.io/video-rvfc/)
+specification.
+The `requestVideoFrameCallback()` method (rVFC) allows web authors to register a callback,
+which runs in the rendering steps when a new video frame 🎞 is sent to the compositor.
+This is intended to allow developers to perform efficient per-video-frame operations on video,
+such as video processing and painting to a canvas, video analysis,
+or synchronization with external audio sources.
+
+## Difference to requestAnimationFrame (rAF)
+
+Operations like drawing a video frame to a canvas via
+[`drawImage()`](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/drawImage)
+made through this API will be synchronized as a best effort
+with the video playing on screen.
+Different from
+[`window.requestAnimationFrame()`](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame),
+which usually fires about 60 times per second,
+`requestVideoFrameCallback()` is bound to the actual video frame rate—with an important
+[exception](https://wicg.github.io/video-rvfc/#ref-for-update-the-rendering③:~:text=Note%3A%20The%20effective%20rate%20at%20which,browser%20would%20fire%20callbacks%20at%2060hz.):
+
+> The effective rate at which callbacks are run is the lesser rate between the video's rate
+  and the browser's rate.
+  This means a 25fps video playing in a browser that paints at 60Hz
+  would fire callbacks at 25Hz.
+  A 120fps video in that same 60Hz browser would fire callbacks at 60Hz.
```

Comment:
What's this quoted from? It needs a sightation.

Reply:
Right before, the linked [exception](https://wicg.github.io/video-rvfc/#ref-for-update-the-rendering③:~:text=Note%3A%20The%20effective%20rate%20at%20which,browser%20would%20fire%20callbacks%20at%2060hz.).

Reply:
What's the reason for quoting this? Can't you just say it in your own words?

Reply:
It's spec text and makes it visually stand out. It invites people to also have a look at the spec if they want to.